server/src/main/resources/prometheus.yml

```
@@ -17,4 +17,9 @@ scrape_configs:
   - job_name: 'eureka-client'
     metrics_path: '/actuator/prometheus'  # Spring Boot Actuator Prometheus 엔드포인트
     static_configs:
-      - targets: ['host.docker.internal:8084', 'host.docker.internal:8089']  # Eureka 클라이언트들의 Prometheus 메트릭 엔드포인트
+      - targets:
+        - 'host.docker.internal:8084'
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 8

(indentation)